src/test/java/uk/gov/mca/beacons/api/controllers/OwnerControllerUnitTest.java

```diff
@@ -64,6 +72,38 @@ void shouldReturn201IfSuccessful() throws Exception {
         .andExpect(status().isCreated());
     }
 
+    @Test
+    void shouldReturn400IfInvalidJsonRequest() throws Exception {
+      final String createOwnerInvalidRequest = new String(
+        Files.readAllBytes(
+          Paths.get(
+            "src/test/resources/fixtures/createOwnerInvalidRequest.json"
+          )
+        )
+      );
+
+      mvc
+        .perform(
+          post("/owner")
+            .contentType(MediaType.APPLICATION_JSON)
+            .content(createOwnerInvalidRequest)
+        )
+        .andExpect(status().isBadRequest())
+        .andExpect(jsonPath("$.errors.length()", is(3)))
+        .andExpect(
+          jsonPath(
+            "$.errors[*].field",
```

Comment:
Checks the field key of all the objects within the `errors` array and asserts that the items on lines 97-99 are present